### PR TITLE
remove all useless header comments

### PR DESCRIPTION
**Pull request by khassel**
see #3358

used command: `find ./ -type f -exec perl -i -0pe 's/\/\*\s*magicmirror.*?\*\/\s*//si' {} \;`

This is a first draft, I think we should preserve some of the comments.

### CODE REVIEW

**Comment by KristjanESPERANTO** [comment]
What do you suggest how we proceed? Comment on the parts where we shouldn't delete everything?

**Comment by khassel** [comment]
I would delete everything expect:

```bash
config/config.js.sample
css/custom.css.sample
modules/default/weather/providers/envcanada.js
modules/default/weather/providers/ukmetofficedatahub.js
modules/default/weather/providers/weathergov.js
tests/configs/modules/calendar/exdate.js
```

In above exceptions we should remove:
- `MagicMirror²`
- `By ...`
- `MIT Licensed.`

**Comment by KristjanESPERANTO** [comment]
Sounds good :+1: I haven't noticed anything else that I would keep.

File: modules/default/weather/providers/yr.js [hunk 1]
```diff
@@ -1,16 +1,5 @@
 /* global WeatherProvider, WeatherObject */
 
-/* MagicMirror²
- * Module: Weather
- * Provider: Yr.no
- *
- * By Magnus Marthinsen
- * MIT Licensed
- *
- * This class is a provider for Yr.no, a norwegian weather service.
```
Comment (rejas) [inline modules/default/weather/providers/yr.js:10]: These weather infos should stay in

File: modules/default/weather/providers/weatherflow.js [hunk 2]
```diff
@@ -1,16 +1,5 @@
...
 
-/* MagicMirror²
- * Module: Weather
- * Provider: Weatherflow
- *
- * By Tobias Dreyem https://github.com/10bias
- * MIT Licensed
- *
- * This class is a provider for Weatherflow.
- * Note that the Weatherflow API does not provide snowfall.
```
Comment (rejas) [inline modules/default/weather/providers/weatherflow.js:11]: These infos should stay in

File: modules/default/weather/providers/weatherbit.js [hunk 3]
```diff
@@ -1,15 +1,5 @@
 /* global WeatherProvider, WeatherObject */
 
-/* MagicMirror²
- * Module: Weather
- * Provider: Weatherbit
- *
- * By Andrew Pometti
- * MIT Licensed
- *
- * This class is a provider for Weatherbit, based on Nicholas Hubbard's class
```
Comment (rejas) [inline modules/default/weather/providers/weatherbit.js:10]: Maybe add a link to weatherbit instead?

File: modules/default/weather/providers/ukmetoffice.js [hunk 4]
```diff
@@ -1,13 +1,5 @@
 /* global WeatherProvider, WeatherObject, WeatherUtils */
 
-/* MagicMirror²
- * Module: Weather
- *
- * By Malcolm Oakes https://github.com/maloakes
- * MIT Licensed.
- *
- * This class is a provider for UK Met Office Datapoint.
```
Comment (rejas) [inline modules/default/weather/providers/ukmetoffice.js:9]: Maybe add a link to the weatherprovider instead?

File: modules/default/weather/providers/smhi.js [hunk 5]
```diff
@@ -1,15 +1,5 @@
 /* global WeatherProvider, WeatherObject */
 
-/* MagicMirror²
- * Module: Weather
- * Provider: SMHI
- *
- * By BuXXi https://github.com/buxxi
- * MIT Licensed
- *
- * This class is a provider for SMHI (Sweden only). Metric system is the only
```
Comment (rejas) [inline modules/default/weather/providers/smhi.js:10]: Should stay in

File: modules/default/weather/providers/pirateweather.js [hunk 6]
```diff
@@ -1,15 +1,5 @@
...
 
-/* MagicMirror²
- * Module: Weather
- * Provider: Pirate Weather
- *
- * Written by Nicholas Hubbard https://github.com/nhubbard for formerly Dark Sky Provider
- * Modified by Karsten Hassel for Pirate Weather
- * MIT Licensed
- *
- * This class is a provider for Pirate Weather, it is a replacement for Dark Sky (same api).
```
Comment (rejas) [inline modules/default/weather/providers/pirateweather.js:11]: should stay in

File: modules/default/weather/providers/openweathermap.js [hunk 7]
```diff
@@ -1,13 +1,5 @@
 /* global WeatherProvider, WeatherObject */
 
-/* MagicMirror²
- * Module: Weather
- *
- * By Michael Teeuw https://michaelteeuw.nl
- * MIT Licensed.
- *
- * This class is the blueprint for a weather provider.
```
Comment (rejas) [inline modules/default/weather/providers/openweathermap.js:9]: Helpful line should stay in

File: modules/default/weather/providers/openmeteo.js [hunk 8]
```diff
@@ -1,15 +1,5 @@
 /* global WeatherProvider, WeatherObject */
 
-/* MagicMirror²
- * Module: Weather
- * Provider: Open-Meteo
- *
- * By Andrés Vanegas
- * MIT Licensed
- *
- * This class is a provider for Open-Meteo, based on Andrew Pometti's class
```
Comment (rejas) [inline modules/default/weather/providers/openmeteo.js:10]: add a link to open meteo instead?

File: modules/default/weather/providers/openweathermap.js (hunk 7 above)
Comment (khassel) [reply modules/default/weather/providers/openweathermap.js:9]: helpful? This is only a copy from `weatherprovider.js`

Comment (khassel) [reply modules/default/weather/providers/openweathermap.js:9]: added useful comment with link instead

Comment (rejas) [reply modules/default/weather/providers/openweathermap.js:9]: ups, my mistake :-)

**Comment by khassel** [comment]
test fails are unrelated, we have Wednesday so the on test will fail today ...

**Review by rejas — APPROVED** [review]

**Pull request merged**